src/components/common/maps/Map.js

```diff
@@ -40,18 +48,27 @@ class SimpleMap extends PureComponent {
       markers = this.getDefaultTargets();
     }
 
-    return markers.map(marker => <CircleMarket key={marker.target.id} lat={marker.target.lat} lng={marker.target.lng} text="" optionsStyle={this.getMarkersOptions(marker.target)} />);
+    return markers.map(marker =>
+      <CircleMarket
+        key={marker.target.id}
+        lat={marker.target.lat}
+        lng={marker.target.lng}
+        text=""
+        optionsStyle={this.getMarkersOptions(marker.target)}
+      />);
   }
 
   /*
     Get the Targets Defaults
   */
   getDefaultTargets() {
-    const markersDefaults = [{ target: { id: 1, lat: -34.8794514, lng: -56.1779299, options: { width: '44px', height: '52px', class: 'markerPoint' } } },
-      { target: { id: 2, lat: -34.8798514, lng: -56.1854299, options: { width: '44px', height: '52px', class: 'markerPoint' } } },
-      { target: { id: 3, lat: -34.8768514, lng: -56.1840299, options: { width: '44px', height: '52px', class: 'markerPoint' } } },
-      { target: { id: 4, lat: -34.8748514, lng: -56.1839299, options: { width: '44px', height: '52px', class: 'markerPoint' } } },
-      { target: { id: 5, lat: -34.8738514, lng: -56.1891299, options: { width: '44px', height: '52px', class: 'markerPoint' } } }
+    const options = { width: '44px', height: '52px', class: 'markerPoint' };
+    const markersDefaults = [
```

Comment:
I think you're ready to remove these. No need to have fake targets when you don't have any on the back-end.

Reply:
It´s done!